specification/deviceprovisioningservices/data-plane/Microsoft.Devices/preview/2021-11-01-preview/device.json

```diff
@@ -0,0 +1,1044 @@
+          "type": "string",
+          "readOnly": true
+        },
+        "deviceCertificateIssuanceSettings": {
+          "$ref": "#/definitions/DeviceCertificateIssuanceSettings",
+          "description": "Certificate issuance settings specification for the device.",
+          "readOnly": true
+        },
+        "deviceHostname": {
+          "description": "Optional configured hostname for the device.",
+          "type": "string",
+          "readOnly": true
+        },
+        "deviceIpAddress": {
+          "description": "Optional configured IP address for the device.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "TpmRegistrationResult": {
+      "description": "TPM registration result.",
+      "type": "object",
+      "properties": {
+        "authenticationKey": {
+          "description": "Encrypted authentication key as a string.",
+          "type": "string"
+        }
+      }
+    },
+    "X509RegistrationResult": {
+      "description": "X509 registration result.",
+      "type": "object",
+      "properties": {
+        "certificateInfo": {
+          "$ref": "#/definitions/X509CertificateInfo"
+        },
+        "enrollmentGroupId": {
+          "type": "string"
+        },
+        "signingCertificateInfo": {
+          "$ref": "#/definitions/X509CertificateInfo"
+        }
+      }
+    },
+    "SymmetricKeyRegistrationResult": {
+      "description": "Registration result returned when using SymmetricKey attestation.",
+      "type": "object",
+      "properties": {
+        "enrollmentGroupId": {
+          "description": "Enrollment group id.",
+          "type": "string",
+          "example": "contoso-enrollment-group"
+        }
+      }
+    },
+    "TrustBundle": {
+      "description": "A collection of trusted root or intermediate certificates associated with one or more device enrollments.",
+      "required": [
+        "certificates"
+      ],
+      "type": "object",
+      "properties": {
+        "certificates": {
+          "description": "The certificates in the trust bundle.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/X509CertificateWithMetadata"
+          }
+        },
+        "id": {
+          "description": "The trust bundle ID. A case-insensitive string (up to 128 characters long) of alphanumeric characters plus certain special characters : . _ -. No special characters allowed at start or end.",
+          "type": "string",
+          "readOnly": true
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "The DateTime this resource was created in UTC.",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastModifiedDateTime": {
+          "format": "date-time",
+          "description": "The DateTime this resource was last updated in UTC.",
+          "type": "string",
+          "readOnly": true
+        },
```

Comment:
Some inconsistency here. In other places you mention "Utc" (unnecessary, as noted) but here "DateTime". We generally encourage "DateTime" as a suffix. Whatever the case, I recommend being consistent throughout.

Reply:
Thanks @heaths, there are legacy API that have this Utc suffix and I'm afraid we are stuck with those. For all the new APIs, I will be making the changes to comply with this format.